lib/modules/platform/gitlab/readme.md

```diff
@@ -41,6 +41,8 @@ Remember to set `platform=gitlab` somewhere in your Renovate config file.
 If you're using a private [GitLab container registry](https://docs.gitlab.com/ee/user/packages/container_registry/), you must:
 
 - Set the `RENOVATE_HOST_RULES` CI variable to `[{"matchHost": "${CI_REGISTRY}","username": "${GITLAB_USER_NAME}","password": "${RENOVATE_TOKEN}", "hostType": "docker"}]`.
+
+  Alternatively, if [`detectHostRulesFromEnv`](../../../../docs/usage/self-hosted-configuration.md#detecthostrulesfromenv) is enabled, you can set the CI variables `DOCKER_REGISTRY_GITLAB_COM_USERNAME=${GITLAB_USER_NAME}` and `DOCKER_REGISTRY_GITLAB_COM_PASSWORD=${RENOVATE_TOKEN}`. (See [issue #17940](https://github.com/renovatebot/renovate/issues/17940))
```

Comment:
```suggestion
  Alternatively, if [`detectHostRulesFromEnv`](../../../../docs/usage/self-hosted-configuration.md#detecthostrulesfromenv) is enabled, you can set the CI variables `DOCKER_REGISTRY_GITLAB_COM_USERNAME=${GITLAB_USER_NAME}` and `DOCKER_REGISTRY_GITLAB_COM_PASSWORD=${RENOVATE_TOKEN}`.
```

I would not link to an issue here, link to the correct docs section instead.

Reply:
Yeah, probably makes sense. But there is no section in the docs yet which explicitly mention and showcase these environment variables AFAIS. So I guess for now the link to `detectHostRulesFromEnv` will do.